### DISCUSSION TITLE
Path or Polygon？

### DISCUSSION THREAD

**Discussion by Zahi28** [HelgeGehring/gdshelpers#45] [General]
Hello，
I find this gdsHelpers very easy to use for describing 2D-masks. But, can I create a path with it? Because I need to use FBMS to get a  long waveguide that's not broken and across several writing fields. For example the length of waveguide is 800um and the writing field range is 100um * 100um.

**Comment by HelgeGehring** [comment]
Hey, nice to hear that you like gdshelpers!
I've recently added support for paths in https://github.com/HelgeGehring/gdshelpers/commit/700cf9af397e8ec6be34234284b33418d6fe4c25, but it was not yet included in the pypi-release. I've just released v1.2.0, where it's also included.
After updating you can use e.g. `cell.add_to_layer(3, LineString(((0, 0), (100, 100))))` to add a Path to your design.
Hope that helps! Please let me know if it works for you.

**Reply by Zahi28** [reply]
With your help, I successfully added a path! But how can I set the width of the path, as shown in the figure below?
![Uploading 屏幕截图 2021-05-31 223630.png…]()

**Reply by Zahi28** [reply]
Sorry, the picture hasn't been uploaded. It looks like this.
![屏幕截图 2021-05-31 223630](https://user-images.githubusercontent.com/37107610/120209751-9e657e80-c261-11eb-9ede-74d5d422d070.png)
